content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -508,7 +508,7 @@ gRPC 服务通过 `/var/lib/kubelet/pod-resources/kubelet.sock` 的 UNIX 套接
 中声明将 `/var/lib/kubelet/pod-resources` 目录以
 {{< glossary_tooltip text="卷" term_id="volume" >}}的形式被挂载到设备监控代理中。
 
-对“PodResourcesLister 服务”的支持要求启用 `KubeletPodResources`
+对 “PodResourcesLister 服务”的支持要求启用 `KubeletPodResources`
```

Comment:
```suggestion
对 “PodResourcesLister 服务” 的支持要求启用 `KubeletPodResources`
```
“” is special as a pair of Chinese quotes and shall be symmetric before and after, which is rendered not very good by Hugo.